### PR TITLE
ReportForm: Add description to the form elements

### PR DESCRIPTION
**Pull request by jhoxhaa**

### CODE REVIEW

File: library/Reporting/Web/Forms/ReportForm.php [hunk 1]
```diff
@@ -34,27 +34,34 @@ protected function assemble()
         $this->setDefaultElementDecorator(new CompatDecorator());
 
         $this->addElement('text', 'name', [
-            'required' => true,
-            'label'    => 'Name'
+            'description' => $this->translate('A unique name of your report. '
+                . 'It will be used when exported to pdf, json or csv format and also when you list the reports in cli'),
```
Comment (yhabteab) [inline library/Reporting/Web/Forms/ReportForm.php:38]: I would rephrase this as follows: 
`A unique name of this report. It is used when exporting to pdf, json or csv format and also when listing the reports in the cli`

**Review by yhabteab — CHANGES_REQUESTED** [review]
In addition to the requested changes, consider doing the following.

- If the method call and the description do not fit on the same line, then it is best to put the description on a separate line. Example:
    ```php
    [
         ... => $this->translate(
             'Description here'
         ),
    ]
   ```
    And when it still doesn't fit in the same line, you can simply wrap to the next line.
- Usually it is better to add the description attribute as the **last one**. If you have e.g. multiline description right at the beginning of the array, then it makes things messy.
   ```php
   [
      Other attributes here,
      'description' => ...
   ]
   ```

File: library/Reporting/Web/Forms/ReportForm.php [hunk 2]
```diff
@@ -34,27 +34,38 @@ protected function assemble()
         $this->setDefaultElementDecorator(new CompatDecorator());
 
         $this->addElement('text', 'name', [
-            'required' => true,
-            'label'    => 'Name'
+            'required'    => true,
+            'label'       => 'Name',
+            'description' => $this->translate(
+                'A unique name of this report. It is used when exporting to pdf, json or csv format'
+                . 'and also when listing the reports in the cli'
```
Comment (yhabteab) [inline library/Reporting/Web/Forms/ReportForm.php:41]: Missing whitespace!

File: library/Reporting/Web/Forms/ReportForm.php [hunk 3]
```diff
@@ -34,27 +34,38 @@ protected function assemble()
...
+            )
         ]);
 
         $this->addElement('select', 'template', [
             'label'   => 'Template',
-            'options' => [null => 'Please choose'] + $this->listTemplates()
+            'options' => [null => 'Please choose'] + $this->listTemplates(),
+            'description' => $this->translate(
+                'Specifies the template to use when exporting this report to pdf. (Default Icinga template)'
+            )
```
Comment (yhabteab) [inline library/Reporting/Web/Forms/ReportForm.php:60]: Please align the double arrow operators `=>` correctly.

**Review by yhabteab — CHANGES_REQUESTED** [review]

**Review by yhabteab — APPROVED** [review]

**Review by nilmerg — CHANGES_REQUESTED** [review]
Please also mark the element and option labels for translation.

**Review by nilmerg — APPROVED** [review]
Nvm. That's what #175 does.
